public/404.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
```

Comment:
This must return a JSON document that represent UI
Just set all to `{ "type": "flex", "children": [{"type": "text", "Error 404: Not found"}] }` for now, this will be implemented later.